### PR TITLE
Create pdf_to_image.py

### PR DESCRIPTION
**Pull request by Anugrahig**
Pdf to image file conversion

### CODE REVIEW

**Comment by Anugrahig** [comment]
Pdf to image conversion

File: AAC/core/pdf_to_image.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Import module 
+from pdf2image import 
+convert_from_path
```
Comment (harisreedhar) [inline AAC/core/pdf_to_image.py]: remove this line

File: AAC/core/pdf_to_image.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+# Store Pdf with Converter_from_path
+def my_function(path):
+  images = convert_from_path('example.pdf')
+  
+  for img in images:
+    img.save('output.jpg','JPEG)
```
Comment (harisreedhar) [inline AAC/core/pdf_to_image.py]: this should be `img.save('output.jpg','JPEG')`

**Review by Anugrahig — COMMENTED** [review]
file is edited

**Comment by Anugrahig** [comment]
pdf_to_image file edited

**Pull request closed without merging**